docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -194,6 +194,16 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
 
 Airgap installations where Rancher is configured to use the packaged copy of Helm system charts ([`useBundledSystemChart=true`](../../../getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md#helm-chart-options-for-air-gap-installations)) will only refer to the [system-chart](https://github.com/rancher/system-charts) repository that comes bundled and will not be able to be refreshed or synced.
 
+#### Refresh Interval 
+
+1. A new field `refreshInterval` has been added to the `ClusterRepo` CRD. The default `refreshInterval` is set to 3600 seconds and Rancher syncs the repository every 3600 seconds for each Helm repository.  
+
+To modify the refresh interval of a chart repository:
+
+1. Find the repository you want to modify, and click **⋮ > Edit YAML**.
```

Comment:
This isn't great UX and I'm not sure we should be pushing it. We show a form when creating the resource and there should be something in there. @rohitsakala can you create an issue in rancher/dashboard to add the field to the form?

Reply:
Ah, hadn't spotted it. Much appreciated, thanks!